File: Data Structures & Algorithms/two-integer-sum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        const res = [];
        let prevMap = new Map();
        for (let i=0; i < nums.length; i++) {
            let diff = target - nums[i]
           
            if (prevMap.has(diff)) {
                
                return [prevMap.get(diff), i]
            } else {
                prevMap.set(nums[i], i)
            }
        }
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        res = 0
        myMax = 0
        for i in range(len(nums)):
            if nums[i] == 1:
                res+=1
            else:
                myMax = max(myMax, res)
                res = 0
            
        return max(res, myMax)